src/drivers/uavcan/sensors/differential_pressure.cpp

```diff
@@ -47,15 +47,28 @@ const char *const UavcanDifferentialPressureBridge::NAME = "differential_pressur
 
 UavcanDifferentialPressureBridge::UavcanDifferentialPressureBridge(uavcan::INode &node) :
 	UavcanSensorBridgeBase("uavcan_differential_pressure", ORB_ID(differential_pressure)),
+	CDev(AIRSPEED0_DEVICE_PATH),
+	_class_instance(-1),
 	_sub_air(node)
 {
 }
 
+UavcanDifferentialPressureBridge::~UavcanDifferentialPressureBridge()
+{
+	if (_class_instance != -1) {
+		unregister_class_devname(AIRSPEED_BASE_DEVICE_PATH, _class_instance);
+	}
+
+}
+
 int UavcanDifferentialPressureBridge::init()
 {
 	// Initialize the calibration offset
 	param_get(param_find("SENS_DPRES_OFF"), &_diff_pres_offset);
 
+	/* register alternate interfaces if we have to */
+	_class_instance = register_class_devname(AIRSPEED_BASE_DEVICE_PATH);
```

Comment:
To register  /dev/airspeed1 for i2c. Otherwise, i2C airspeed cannot be calibrated after CAN airspeed is connected.Because the CAN airspeed occupies dev/airspeed0.